deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,117 @@
+Install Spark .NET Interactive Notebook Experiences on your HDInsight Spark Cluster
+===
+
+# Table of Contents
+This documentation will describe how to use Spark .NET within Jupyter Notebooks available on your HDInsight Spark Clusters.
+- [Background](#background)
+- [Pre-requisites](#pre-requisites)
+- [Installation Procedure](#installation-procedure)
+  - [Step 1. Stop Livy Server](#step-1-stop-livy-server)
+  - [Step 2. Submit HDInsight Script Action](#step-2-submit-hdinsight-script-action)
+  - [Step 3. Start Livy Server](#step-3-start-livy-server)
+  - [Step 4. Set up Spark Default Configs](#step-3-start-livy-server)
+- [Submit Jobs through Jupyter Notebook](#submit-jobs-through-jupyter-notebook)
+
+## Background
+To provide the ability to use Spark .NET within your Jupyter Notebooks, an important pre-requisite is a C# REPL that is capable of executing your C# code line-by-line (and preserving execution state, where needed). We have integrated [dotnet-try](https://github.com/dotnet/try) as our official dotnet REPL to provide you with the best experience.
+
+To provide the end-to-end integration, we had to add a Jupyter kernel along with necessary modifications to [Apache Livy](https://github.com/apache/incubator-livy) and [sparkmagic](https://github.com/jupyter-incubator/sparkmagic). Since these code changes have not yet been merged into the respective open source projects, to be able to get the out-of-box Spark .NET experience, you will have to patch components in your HDInsight Spark Cluster with the binaries we provide. Please note that this is *experimental* and not supported by the HDInsight Spark team. *Use it at your own risk*. Please do remember that we are working hard to get these changes into the respective projects so you do not need these manual steps.
```

Comment:
I feel the disclaimer should be added at the end of the Background section with something like 

> **Note**: disclaimer

Reply:
How about the below format?
```suggestion
To provide the end-to-end integration, we had to add a Jupyter kernel along with necessary modifications to [Apache Livy](https://github.com/apache/incubator-livy) and [sparkmagic](https://github.com/jupyter-incubator/sparkmagic). Since these code changes have not yet been merged into the respective open source projects, to be able to get the out-of-box Spark .NET experience, you will have to patch components in your HDInsight Spark Cluster with the binaries we provide. 
> **Disclaimer:**: Please note that this is *experimental* and not supported by the HDInsight Spark team. *Use it at your own risk*. Please do remember that we are working hard to get these changes into the respective projects so you do not need these manual steps.

```

Reply:
Placed after the next paragraph.

Reply:
Changed